src/shared/actions/transfers.js

```diff
@@ -874,8 +903,19 @@ export const retryFailedTransaction = (accountName, bundleHash, powFn) => (dispa
                 // If all addresses are still unspent, retry
                 return retry()(
                     existingFailedTransactionsForThisAccount[bundleHash],
-                    // If proof of work is set to remote, pass in null as the proof of work function
-                    shouldOffloadPow ? null : powFn,
+                    // If proof of work configuration is set to remote,
+                    // Extend seedStore object with offloadPow
+                    // This property will lead to perform network bound proof-of-work
```

Comment:
```suggestion
                    // This property will lead to perform remote proof-of-work
```

Reply:
@rajivshah3 Fixed in https://github.com/iotaledger/trinity-wallet/pull/775/commits/aa685de00950428a514623fb6d7abab48ff21b34